File: leetcode_150/array/remove_duplicates_from_sorted_array_26.java
class Solution {
    public int removeDuplicates(int[] nums) {
        var k = 1;
        if (nums.length < 2) {
            return k;
        }
        for (int i = 1; i < nums.length; i++) {
            if (nums[i] > nums[i - 1]) {
                nums[k] = nums[i];
                k++;
            }
        }
        return k;
    }
}

File: leetcode_150/array/remove_duplicates_from_sorted_array_26.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        k = 1
        if len(nums) < 2:
            return k
        for i in range(1, len(nums)):
            if nums[i] > nums[i-1]:
                nums[k] = nums[i]
                k+=1
        return k
        